core-bundle/src/Resources/config/services.yml

```diff
@@ -106,7 +106,11 @@ services:
         tags:
             - controller.service_arguments
 
-    Contao\CoreBundle\Controller\FrontendController: ~
+    Contao\CoreBundle\Controller\FrontendController:
+        arguments:
+            - '@Contao\CoreBundle\Cron\Cron'
```

Comment:
I think this argument is wrong? The cron service is injected on the action, not the constructor.

Reply:
Fixed in 80a2dab86